packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/drawable/CSSGradient.java

```diff
@@ -0,0 +1,74 @@
+package com.facebook.react.uimanager.drawable;
```

Comment:
1. We're wanting all new code to be Kotlin instead of Java.
2. Could we put this in `com.facebook.react.uimanager.style` instead of `com.facebook.react.uimanager.drawable`? Potentially just named something like `Gradient`.